command/host_create.go

```diff
@@ -125,9 +128,14 @@ func (c *createHost) waitForLogs(ctx context.Context, comm client.Communicator,
 			batchEnd := time.Now()
 			status, err := comm.GetDockerStatus(ctx, hostID)
 			if err != nil {
-				return errors.Wrapf(err, "error getting docker status")
+				grip.Info(message.WrapError(err, "problem receiving docker logs in host.create"))
+				if startedCollectingLogs {
+					return nil // container has likely exited
```

Comment:
Is there another way to tell that this has happened? I'm concerned that one failure to retrieve logs will cause the client to stop trying to get logs. (Potentially this is another ticket.)